module/utils.mjs

```diff
@@ -426,10 +454,44 @@ export function getSceneTargets() {
 /*  Conversions                                 */
 /* -------------------------------------------- */
 
+/**
+ * Convert the provided time value to another unit. If not final unit is provided, then will convert it to the largest
```

Comment:
```suggestion
 * Convert the provided time value to another unit. If ot final unit is provided, then will convert it to the largest
```

Reply:
I think you deleted the wrong letter there :)